lambda/syncprocessor/lambda_function.py

```diff
@@ -87,12 +87,17 @@ def processImage(documentId, features, bucketName, outputBucketName, objectName,
     print("path: " +  path)
     maxPages = 100
     comprehendClient = ComprehendHelper()
-    processedComprehendData = comprehendClient.processComprehend(outputBucketName, 'response.json', path, maxPages)
+    comprehendandMedicalEntities = comprehendClient.processComprehend(outputBucketName, 'response.json', path, maxPages)
```

Comment:
Can we follow the camelCase across to have comprehendAndMedicalEntities?

Reply:
Okay sure. Will change it back.